examples/knowledge_graph/sree/relations.csv

```diff
@@ -0,0 +1,10 @@
+pageURL,originaltext,subject,object ,verb
+http://epirecip.es/epicookbook/chapters/sir/intro,The susceptible-infected-recovered (SIR) model in a closed population,susceptible-infected-recovered (SIR) model,closed population,in
+http://epirecip.es/epicookbook/chapters/sir/intro,The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick,susceptible-infected-recovered (SIR) model,Kermack and McKendrick,proposed
+http://epirecip.es/epicookbook/chapters/sir/intro,The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick as a special case,susceptible-infected-recovered (SIR) model,special case,as
+http://epirecip.es/epicookbook/chapters/sir/intro,"The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick as a special case of a more general model, ",susceptible-infected-recovered (SIR) model,general ,of
+http://epirecip.es/epicookbook/chapters/sir/intro,"The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick as a special case of a more general model, and forms the framework",susceptible-infected-recovered (SIR) model,framework,forms
+http://epirecip.es/epicookbook/chapters/sir/intro,"The susceptible-infected-recovered (SIR) model in a closed population was proposed by Kermack and McKendrick as a special case of a more general model, and forms the framework of many compartmental models. ",susceptible-infected-recovered (SIR) model,compartmental models,forms
+http://epirecip.es/epicookbook/chapters/sir/intro,"Susceptible individuals, S, are infected by infected individuals, I, at a per-capita rate βI, and infected individuals recover at a per-capita rate γ to become recovered individuals, R.",susceptible ,infected ,infeced by
+http://epirecip.es/epicookbook/chapters/sir/intro,"Susceptible individuals, S, are infected by infected individuals, I, at a per-capita rate βI, and infected individuals recover at a per-capita rate γ to become recovered individuals, R.",infected,infect,at

```

Comment:
Should probably be something like: `susceptible individuals, infected individuals, are infected by`